src/data/items.h

```diff
@@ -10315,7 +10315,7 @@ const struct Item gItems[] =
     [ITEM_REINS_OF_UNITY] =
     {
         .name = _("ReinsOfUnity"),
-        .pluralName = _("ReinsOfUnities"),
+        .pluralName = _("ReinsOfUnity"),
```

Comment:
If it's the same, why not just remove it?

Reply:
Because not setting pluralName would just append -s and you'd end up with "Reins of Unitys" which is even less correct than what it used to be :D